scripts/nlp_api_master.py

```diff
@@ -169,6 +226,77 @@
+                        "keras_nlp.models.DistilBertMaskedLM",
+                        "keras_nlp.models.DistilBertMaskedLM.from_preset",
+                        "keras_nlp.models.DistilBertMaskedLM.backbone",
+                        "keras_nlp.models.DistilBertMaskedLM.preprocessor",
+                    ],
+                },
+                {
+                    "path": "distil_bert_masked_lm_preprocessor",
+                    "title": "DistilBertMaskedLMPreprocessor layer",
+                    "generate": [
+                        "keras_nlp.models.DistilBertMaskedLMPreprocessor",
+                        "keras_nlp.models.DistilBertMaskedLMPreprocessor.from_preset",
+                        "keras_nlp.models.DistilBertMaskedLMPreprocessor.tokenizer",
+                    ],
+                },
+            ],
+        },
+        {
+            "path": "gpt2/",
+            "title": "GPT2",
+            "toc": True,
+            "children": [
+                {
+                    "path": "gpt2_tokenizer",
+                    "title": "GPT2Tokenizer",
+                    "generate": [
+                        "keras_nlp.models.GPT2Tokenizer",
+                        "keras_nlp.models.GPT2Tokenizer.from_preset",
+                    ],
+                },
+                {
+                    "path": "gpt2_preprocessor",
+                    "title": "GPT2Preprocessor layer",
+                    "generate": [
+                        "keras_nlp.models.GPT2Preprocessor",
+                        "keras_nlp.models.GPT2Preprocessor.from_preset",
+                        "keras_nlp.models.GPT2Preprocessor.tokenizer",
+                    ],
+                },
+                {
+                    "path": "gpt2_backbone",
+                    "title": "GPT2Backbone model",
+                    "generate": [
+                        "keras_nlp.models.GPT2Backbone",
+                        "keras_nlp.models.GPT2Backbone.from_preset",
+                    ],
+                },
+                {
+                    "path": "gpt2_causal_lm",
+                    "title": "GPT2CausalLM model",
+                    "generate": [
+                        "keras_nlp.models.GPT2CausalLM",
+                        "keras_nlp.models.GPT2CausalLM.from_preset",
+                        "keras_nlp.models.GPT2CausalLM.generate",
+                        "keras_nlp.models.GPT2CausalLM.backbone",
+                        "keras_nlp.models.GPT2CausalLM.preprocessor",
+                    ],
+                },
+                {
+                    "path": "gpt2_causal_lm_preprocessor",
+                    "title": "GPT2CausalLMPreprocessor layer",
+                    "generate": [
+                        "keras_nlp.models.GPT2CausalLMPreprocessor",
+                        "keras_nlp.models.GPT2CausalLMPreprocessor.from_preset",
+                        "keras_nlp.models.GPT2CausalLMPreprocessor.tokenizer",
```

Comment:
Shortly we might want to add `generate_preprocess` and `generate_postprocess` here.